Readme.md

```diff
@@ -1 +1,2 @@
 This is the branch for creating features of receivers.
```

Comment:
Readme.md should contain, how to set-up any dependencies (e.g. spark) if required, how to run your code (commands), how to run your test suite (expected outputs)